python/lib/cloudutils/serviceConfig.py

```diff
@@ -617,11 +617,8 @@ def config(self):
             cfo.addEntry("LIBVIRTD_ARGS", "-l")
             cfo.save()
             if os.path.exists("/lib/systemd/system/libvirtd.socket"):
-                bash("/bin/systemctl mask libvirtd.socket");
-                bash("/bin/systemctl mask libvirtd-ro.socket");
-                bash("/bin/systemctl mask libvirtd-admin.socket");
-                bash("/bin/systemctl mask libvirtd-tls.socket");
-                bash("/bin/systemctl mask libvirtd-tcp.socket");
+                bash("/bin/systemctl mask libvirtd.socket libvirtd-ro.socket libvirtd-admin.socket libvirtd-tls.socket libvirtd-tcp.socket");
+                bash("/bin/systemctl mask virtqemud.socket virtqemud-ro.socket virtqemud-admin.socket virtqemud virtnetworkd virtstoraged");
 
```

Comment:
@rohityadavcloud 
the line 619 ensures the services exist.
```
            if os.path.exists("/lib/systemd/system/libvirtd.socket"):
                bash("/bin/systemctl mask libvirtd.socket libvirtd-ro.socket libvirtd-admin.socket libvirtd-tls.socket libvirtd-tcp.socket");
```

a similar check may be needed for the new services
```
                bash("/bin/systemctl mask virtqemud.socket virtqemud-ro.socket virtqemud-admin.socket virtqemud virtnetworkd virtstoraged");
```

Reply:
These are guarded by that condition, aren´t they, @weizhouapache ?
cc @rohityadavcloud
If it is the intention to mask all if "/lib/systemd/system/libvirtd.socket" exists, this should work

Reply:
Yes, the notion is correct. We want to mask is the libvirtd.socket exists. @shwstppr can help double check this @weizhouapache

Reply:
Done @weizhouapache

Reply:
looks good @rohityadavcloud 
thanks for update